content/technical-guide/installation/supported-environments.md

```diff
@@ -10,15 +10,20 @@ menu:
 
 The following environments are supported:
 
-# Server
+# Operating System
 
-- Linux operating system on `amd64` architecture
+- server installations (production setup):
```

Comment:
Would be nice to have links to the installation guides for `production setup` and `demo setup`. WDYT?